pkg/platform/model/inventory.go

```diff
@@ -83,32 +93,56 @@ func FetchAuthors(ingredID, ingredVersionID *strfmt.UUID) (Authors, error) {
 	return results.Payload.Authors, nil
 }
 
-func searchIngredientsNamespace(limit int, ns Namespace, name string) ([]*IngredientAndVersion, error) {
-	lim := int64(limit)
+func searchIngredientsNamespace(ns Namespace, name string, includeVersions bool) ([]*IngredientAndVersion, error) {
```

Comment:
I modified this function to make use of pagination as well as to add a param allowing you to include version variants of the ingredients or just the base ingredient itself.